src/components/registerForm.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+    <div class="mx-auto p-12 rounded-[50px] w-min bg-primary-600">
+        <div class="p-4">
+            <h1 class="text-3xl font-medium mb-1">{{ $t("auth.register") }}</h1>
+            <p>{{ $t("auth.fillOutYourInfo") }}</p>
+            <div class="mt-12">
+                <form class="space-y-5">
+                    <div class="w-96">
+                        <BaseInput id="name" type="text" v-model="name" :error-text="nameErrorMessage" :place-holder="$t('auth.name')" :auto-focus="true"></BaseInput>
+                    </div>
+                    <div class="w-96">
+                        <BaseInput id="email" type="email" v-model="email" :error-text="emailErrorMessage" :place-holder="$t('auth.email')"></BaseInput>
+                    </div>
+                    <div class="w-96">
+                        <BaseInput id="password" type="password" v-model="password" :error-text="passwordErrorMessage" :place-holder="$t('auth.password')"></BaseInput>
+                    </div>
+                    <div class="w-96">
+                        <BaseInput id="passwordConfirm" type="password" v-model="confirmPassword" :error-text="passwordConfirmErrorMessage" :place-holder="$t('auth.confirmPassword')"></BaseInput>
+                    </div>
+                    <div>
+                        <p class="text-red-600 w-full ml-2 mb-2 pt-4 text-xs">{{ errorMessage }}</p>
+                        <BaseButton type="submit" @click="register">{{ $t("auth.register") }}</BaseButton>
+                    </div>
+                    <div>
+                        <p class="text-primary-300 font-bold w-full mb-4 text-xs" v-html="$t('auth.alreadyHaveAnAccount')"></p>
+                    </div>
+                </form>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import BaseInput from "./baseInput.vue";
+import BaseButton from "./baseButton.vue";
+
+export default {
+    components: {
+        BaseInput, BaseButton
+    },
+    data() {
+        return {
+            name: "",
+            email: "",
+            password: "",
+            confirmPassword: "",
+            errorMessage: "",
+            nameErrorMessage: "",
+            emailErrorMessage: "",
+            passwordErrorMessage: "",
+            passwordConfirmErrorMessage: ""
+        }
+    },
+    computed: {
+        doesPasswordMatch() {
+            return this.password === this.confirmPassword;
+        }
+    },
+    methods: {
+        register(): void {
+            if (!this.doesPasswordMatch) {
+                this.errorMessage = this.$t("auth.passwordsIdentical");
```

Comment:
Maybe change this to another errormessage, perhaps the passwordErrorMessage and/or passwordConfirmErrorMessage

Reply:
Changes added